src/transformers/trainer.py

```diff
@@ -465,102 +467,12 @@ def create_optimizer_and_scheduler(self, num_training_steps: int):
                 self.optimizer, num_warmup_steps=self.args.warmup_steps, num_training_steps=num_training_steps
             )
 
-    def setup_wandb(self):
```

Comment:
Removing this method is the only real breaking change, but I doubt we have a lot of users that subclassed Trainer to implement a custom `setup_wandb`. For those users, the new way is to subclass `WandbCallback` and override the `setup` method.

Reply:
Two breaking changes: `setup_comet()` was removed as well. But that is okay with me too. Just wanted to point that out if this is listed in a set of "breaking changes" in the release notes.